pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -123,6 +123,17 @@ def test_loc_multiindex_missing_label_raises(self):
         with pytest.raises(KeyError, match=r"^2$"):
             df.loc[2]
 
+        # Lists with missing labels do not raise:
+        expected = df.iloc[[0, 1]]
+        result = df.loc[[2, 4]]
+        tm.assert_frame_equal(result, expected)
+
+        # ... even when none is found:
+        expected = df.iloc[[]]
+        for l in [2], [2, 3]:  # GH 27148
```

Comment:
better to use parametrisation than a for loop.

Reply:
(related to above)

Reply:
I agree here, would be nice to parameterize this, easier to grok; no big deal to copy-paste the setup (no fixture really needed here)